src/Microsoft.ApiDesignGuidelines.Analyzers/Core/IdentifiersShouldNotContainTypeNames.cs

```diff
@@ -118,5 +125,10 @@ private static void AnalyzeSymbol(ISymbol symbol, SymbolAnalysisContext context)
                 context.ReportDiagnostic(diagnostic);
             }
         }
+
+        private static bool IsObjectEqualsOverride(ISymbol symbol)
```

Comment:
This will ignore all methods with name Equals. I thought we had an IMethodSymbolExtensions method in AnalyzerUtilities for this. Can you check?